package.json

```diff
@@ -0,0 +1,42 @@
+{
+  "name": "math-magicians",
+  "private": true,
+  "version": "0.0.0",
+  "type": "module",
+  "scripts": {
+    "dev": "vite",
+    "build": "vite build",
+    "preview": "vite preview",
+    "stylelint": "stylelint \"**/*.{css,scss}\"",
+    "eslint": "eslint \"**/*.{js,jsx}\"",
+    "fix": "eslint \"*/*.{js,jsx}\" --fix && stylelint \"**/*.{css,scss}\" --fix"
+  },
```

Comment:
- [ ] According to the requirements the project should have been built using the Create React app command, please look at [this article](https://create-react-app.dev/) to see how that works.